pandas/io/pytables.py

```diff
@@ -4140,6 +4145,17 @@ def write(
             # set the table attributes
             table.set_attrs()
 
+            if track_times is not None:
+                from tables import __version__ as tables_version
+                from packaging.version import Version
+
+                if Version(tables_version) >= Version("3.4.3"):
```

Comment:
use LooseVersion

Reply:
changed. Thanks for hint. But, what is the reason for using LooseVersion? I found that `packaging.version` is more "modern" - https://stackoverflow.com/a/11887885/1386342